README.md

```diff
@@ -38,6 +38,7 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
 * [Vessel](https://github.com/slawrence/vessel) - A "tunnel" game written in Elm!
 * [Sliding Puzzle](https://github.com/moroshko/sliding-puzzle) - Configurable sliding puzzle game written in Elm.
 * [TodoMVC](https://github.com/evancz/elm-todomvc) - Proper implementation of the TodoMVC app.
+* [TodoMVC with JSON API](https://github.com/andrewsuzuki/elm-todo-rest-api) - Bare-bones, modular, heavilty-documented todo app with JSON API persistence.
```

Comment:
@lwm thanks, fixed in [b2cce6cbc0ff837838a8e9344c182a4b55a31e4f](https://github.com/isRuslan/awesome-elm/commit/b2cce6cbc0ff837838a8e9344c182a4b55a31e4f)